composer.json

```diff
@@ -12,7 +12,7 @@
   "require": {
     "php": ">=7.1",
     "auto1-oss/service-api-request": "^1.0",
-    "auto1-oss/service-api-components-bundle": "dev-symfony-6-support",
+    "auto1-oss/service-api-components-bundle": "^1.0",
```

Comment:
I guess it should be `^1.2.0` as this is lowest supported version of the dependency, right?

Reply:
Also though about that. But there were no changes in `auto1-oss/service-api-components-bundle` which would require min version to be updated, so it should work with versions lower than `1.2.0`. Only if we want to force update `auto1-oss/service-api-components-bundle` for everyone with new version of `service-api-handler-bundle`.

Reply:
yes, i would also keep it as it is